docs/README.md

```diff
@@ -0,0 +1,12 @@
+## Learn More
+
+* [Contribute](CONTRIBUTING.md)
+* [License](../LICENSE.md)
+
+### Resources
+
+* [GitHub Repository](https://github.com/src-d/sourced-ce)
+* [Product Page](https://www.sourced.tech)
+* [Book a Demo](https://go.sourced.tech/engine)
```

Comment:
I don't think this will be the final link. Better to leave it out, or just remove the link, than linking to a wrong URL in my opinion.

Reply:
It's already used in the new landing. I guess that @vcoisne will provide a new link if needed, or redirect to the new one if he prefers so.

Anyhow, he already changed form content, so now it relates to our new product.